src/main.rs

```diff
@@ -33,7 +34,7 @@ pub struct Cli {
     eval: Option<String>,
 
     /// Generate docs
-    /// (OUTPUT is dir instead, default: `docs/`)
+    /// (OUTPUT is dir instead, default: `docs/` if missing it will generate the folder)
```

Comment:
Isn't it a standard behavior?

Reply:
yeah but it isn't clear, usually cli tools doesn't create folder expect that already exists.
it is better to inform